### PR TITLE
Add missing semicolons and add a subDir = false

### PR DESCRIPTION
**Pull request by Gerhard-Kanzler**

### CODE REVIEW

**Comment by g105b** [comment]
Thanks for the PR @Gerhard-Kanzler. Please can you add a test for the new subDir functionality, then I'll get it merged in.

**Comment by Gerhard-Kanzler** [comment]
So now i added the tests :)

**Comment by g105b** [comment]
Thanks for this. Sorry for delaying but I can now accept this once the merge conflicts have been solved. Is it possible for you to look into this?

**Comment by Gerhard-Kanzler** [comment]
Ok now you can merge :)

File: bower-installer.spec.js [hunk 1]
```diff
@@ -33,7 +33,9 @@ function expectFilesToExist(files, run) {
 		'test/glob/build',
 		'test/glob/bower_components',
 		'test/keyword/build',
-		'test/keyword/bower_components'
+		'test/keyword/bower_components',
+        'test/moveToSubDir/build',
```
Comment (dnalot) [inline bower-installer.spec.js]: Wrong style tabs.

File: lib/installer.js [hunk 2]
```diff
@@ -53,24 +52,28 @@ function installFile(f, pakcfg, paths, dep, silent, options, callback) {
     // If the configured paths is a map, use the path for the given file extension
     if (paths.all) {
 
-        path = utils.parsePath(paths.all, pakcfg) + pathSep + key + subDir;
+        path = utils.parsePath(paths.all, pakcfg) + ( pakcfg.noSubDir === true ? '' : (pathSep + key) ) + subDir;
```
Comment (dnalot) [inline lib/installer.js]: Good catch. It should also be checking `utils.hasVariableInPath`. Just PRed #140.

**Comment by dnalot** [comment]
None of your `.idea` files should be committed. Add it to your global ignore.

File: lib/installer.js [hunk 3]
```diff
@@ -53,24 +52,28 @@ function installFile(f, pakcfg, paths, dep, silent, options, callback) {
...
         if (path && typeof path !== 'undefined') {
 
             if (!utils.hasVariableInPath(path)) {
-                path += (options.includePackageNameInInstallPath ? pathSep + key : '') + subDir; 
-            }
-            else {
+                if( pakcfg.noSubDir === true ){
+                    path += pathSep + subDir;
+                }else{
+                    path += (options.includePackageNameInInstallPath ? pathSep + key : '') + subDir;
```
Comment (dnalot) [inline lib/installer.js]: Should this be changed to your config too?

**Pull request closed without merging**